ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,21 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-efa-pg-f28
+  cron: 32 15 5 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      ENABLE_OTA_TEST: OCP-23799
+    test:
+    - ref: cucushift-upgrade-healthcheck
+    - ref: cucushift-upgrade-setchannel
+    - ref: cucushift-ota-preupgrade
+    - ref: cucushift-upgrade-toversion
+    - ref: cucushift-ota-postupgrade
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-aws-ipi-efa-pg
```

Comment:
If this case is only to cover --tolatest upgrade option, we can use a simple install workflow, e.g: aws-ipi

Reply:
I am choosing this profile is because its a new profile in e2e but have not covered in upgrade ci:
![image](https://github.com/openshift/release/assets/83053288/7b620491-d918-41d2-97be-685c37190e90)

Reply:
If ci jobs is purely for OTA test case automation usage, it is not a part of regular ci matrix. If you want to cover it as a part of regular upgrade ci, we should add a dedicated ci job in nightly upgrade ci, but that is out of this PR scope.

Reply:
Sure